.screenrc

```diff
@@ -0,0 +1,5 @@
+sessionname "bs-containers test console"
```

Comment:
The whole file? That's intentional. It's the configuration file for GNU screen, a terminal multiplexer similar to tmux, and allows you to run both `bsb` and `ava` watchers in a single terminal. Try installing `screen` and run `npm run watch:screen`.